js/repl/AccordionTab.js

```diff
@@ -8,49 +8,31 @@ import { colors, media } from "./styles";
 type Props = {
   children: React$Element<any>,
   className: ?any,
-  defaultExpanded?: boolean,
+  isExpanded?: boolean,
   label: string,
+  toggleIsExpanded: Function,
```

Comment:
Very minor nit: Call it `onExpandedToggle` or something like that? At least in my team at Facebook, we prefix all callbacks with `on`. Not sure if that's common or not. If it's not common in open source, feel free to ignore this comment.

Reply:
I think this is a fairly common convention, and one I normally follow too- but only for callbacks that receive event-like arguments. In this case, since the method is actually just a _toggle_ (eg it doesn't accept any parameters) I thought the "on" prefix would be misleading.